docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md

```diff
@@ -1,24 +1,27 @@
 # KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert
+<!-- Edited: apinnick@redhat.com, Aug 2022 --> 
 
 ## Meaning
-The `KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert` alert means that the HypeConverged cluster operator (HCO) is running, but the `HyperConverged` custom resource does not exist for more than an hour.
+This alert fires when the HyperConverged Operator (HCO) runs for more than an hour without a `HyperConverged` custom resource (CR).
```

Comment:
Please revert `HyperConverged Operator` to `HypeConverged cluster operator`

Reply:
Thanks. I checked the [repo](https://github.com/kubevirt/hyperconverged-cluster-operator). I will add that to the "Guidelines" and update this PR.